packages/polywrap-client/tests/test_bignumber.py

```diff
@@ -0,0 +1,108 @@
+    result = await client.invoke(options)
+    print(result)
+    assert result.result == "123000"
+
+async def test_invoke_bignumber_with_1arg_and_2props():
+    client = PolywrapClient()
+    # BigNumber wrapper schema - https://wrappers.io/v/ipfs/Qme2YXThmsqtfpiUPHJUEzZSBiqX3woQxxdXbDJZvXrvAD
+    uri = Uri(f'fs/{Path(__file__).parent.joinpath("cases", "simple-invoke" ,"wrapperBigNumber.wasm").absolute()}')
+    args = {
+        "arg1": "123123",
+        "obj": {
+            "prop1": "1000",
+           "prop2": "4"
+        }
+    }
+    options = InvokerOptions(uri=uri, method="method", args=args, encode_result=False)
+    result = await client.invoke(options)
+    print(result)
+    assert result.result == str(123123*1000*4)
+
+async def test_invoke_bignumber_with_2args_and_1prop():
+    client = PolywrapClient()
+    # BigNumber wrapper schema - https://wrappers.io/v/ipfs/Qme2YXThmsqtfpiUPHJUEzZSBiqX3woQxxdXbDJZvXrvAD
+    uri = Uri(f'fs/{Path(__file__).parent.joinpath("cases", "simple-invoke" ,"wrapperBigNumber.wasm").absolute()}')
+    args = {
+        "arg1": "123123",
+        "obj": {
+            "prop1": "1000",
+           "prop2": "444"
+        }
+    }
+    options = InvokerOptions(uri=uri, method="method", args=args, encode_result=False)
+    result = await client.invoke(options)
+    print(result)
+    assert result.result == str(123123*1000*444)
+
+async def test_invoke_bignumber_with_2args_and_2props():
+    client = PolywrapClient()
+    # BigNumber wrapper schema - https://wrappers.io/v/ipfs/Qme2YXThmsqtfpiUPHJUEzZSBiqX3woQxxdXbDJZvXrvAD
+    uri = Uri(f'fs/{Path(__file__).parent.joinpath("cases", "simple-invoke" ,"wrapperBigNumber.wasm").absolute()}')
+    args = {
+        "arg1": "123123",
+        "arg2": "555",
+        "obj": {
+            "prop1": "1000",
+           "prop2": "4"
+        }
+    }
+    options = InvokerOptions(uri=uri, method="method", args=args, encode_result=False)
+    result = await client.invoke(options)
+    print(result)
+    assert result.result == str(123123*555*1000*4)
+
+async def test_invoke_bignumber_with_2args_and_2props_floats():
+    client = PolywrapClient()
+    # BigNumber wrapper schema - https://wrappers.io/v/ipfs/Qme2YXThmsqtfpiUPHJUEzZSBiqX3woQxxdXbDJZvXrvAD
+    uri = Uri(f'fs/{Path(__file__).parent.joinpath("cases", "simple-invoke" ,"wrapperBigNumber.wasm").absolute()}')
+    print(Uri)
+    args = {
+        "arg1": "123.123",
+        "arg2": "55.5",
+        "obj": {
+            "prop1": "10.001",
+           "prop2": "4"
+        }
+    }
+    options = InvokerOptions(uri=uri, method="method", args=args, encode_result=False)
+    result = await client.invoke(options)
+    print(result)
+    assert result.result == str(123.123*55.5*10.001*4)
+
+
+# async def test_invoke_bignumber_with_2args_and_2props_complex_numbers():
+#     client = PolywrapClient()
+#     # BigNumber wrapper schema - https://wrappers.io/v/ipfs/Qme2YXThmsqtfpiUPHJUEzZSBiqX3woQxxdXbDJZvXrvAD
```

Comment:
No polywrap doesn't support complex numbers yet and isn't planning to do so anytime soon.

Reply:
I don't see a real need for this yet but correct me if I am wrong